docs/configuration/large_language_model.md

```diff
@@ -88,21 +88,59 @@ ludwig train ...
 
 ## Input Features
 
-Currently, the LLM model type only supports a single input feature of type `text`.
+Because of the way LLMs work, their input is always a single chunk of `text`. But this input can be 
+built up from one or more columns of the input dataset and optional interspersed static text. 
+The following two examples illustrate the range of possibilities.
 
-If no `prompt` template is provided, this feature must correspond to a column
-in the input dataset. If a prompt template is provided, the rendered prompt
-will be used as the input feature value during training and inference.
+### Single Dataset Column
+
+If input to the LLM is just the content of a single dataset column (without any other prefixed or
+suffixed text), no `prompt` template should be provided and the `name` of the feature must 
+correspond to a column in the input dataset. See the following example.
```

Comment:
@sanjaydasgupta likely, I would say here "and the `name` of the feature must correspond to that specific database column".

Reply:
I took the liberty of changing your `database` to `dataset` for the sake of consistency throughout the document